### DISCUSSION TITLE
side by side editing in 11ty etc

### DISCUSSION THREAD

**Discussion by storynory** [tinacms/tinacms#3873] [Ideas]
I'm using 11ty. it's not reactive, but it does have hot-reload in developer mode.  It would be nice to have side-by side editing instead of "nothing to edit here" in the side panel.  It should reload on every save.  Is this possible?

**Comment by logan-anderson** [comment]
@storynory Not possible right now. The forms in the sidebar are loaded by calling the `useTina` hook which is only available in react. Probably best to stick with separate tabs for now (They could be side by side).

This is sorta a hack but you could try open a new tab in the router property on a collection (When in local dev)

```ts
ui: {
   router: ({ document }) => {
          // replace localDev with some metric to see if your running locally. Could be process.env.NODE_ENV === 'development'
           if(LocalDev) {
               // open link in a new tab
               window.open(url, '_blank').focus();
           }
           return null;
   },
}
```
